platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -121,6 +123,8 @@ public static String configureDemoCert(String commonName) {
   public static String configureDemoCert(String commonName, @Nullable String[] subjectAltNames) {
     CertificateSigningRequest csr = new CertificateSigningRequest();
     csr.setCommonName(commonName);
+    // Required for Chrome support (DDF-3104)
+    csr.addSubjectAlternativeNames(String.format("DNS:%s", commonName));
```

Comment:
❓ Should we check if the same SAN was provided first?

Reply:
The docs should probably be updated to indicate that this is an idempotent operation as we maintain the sans in a set. Adding the same name twice won't have any effect.

Reply:
Per @coyotesqrl's comment below, I think the SANs are maintained in a set so no duplication will occur.

Reply:
🤦‍♂️ I added that code